### ISSUE TITLE
Bug: Inconsistent Color Rgb behaviour

### ISSUE THREAD

**Issue by DarioDaF** [pixijs/pixijs#9377]
### Current Behavior

`new PIXI.Color({r: 1.0, g: 0.0, b: 0.0}).toRgb()` returns `{ "r": 0.003921568859368563, "g": 0, "b": 0 }`

### Expected Behavior

`RgbColor` source and `RgbColor` color outputs should match and use the same convention (expecially since they are the same type in the TS stubs)

### Steps to Reproduce

Just run the code in current behaviour in the browser

### Environment

- **`pixi.js` version**: *7.2.4*
- **Browser & Version**: *Chrome 112*
- **OS & Version**: *Windows 11*

### Possible Solution

The 2 possible solutions are the obvious:
- Make the source read as normalized colors or
- Make the `toRgb` output be `u8t`

The second one doesn't seem a good option for resolution reasons.

If none of those are reasonable due to compatibility etc... at least change to a more descriptive name the `ColorSource` union types to be something like `Rgb[a]U8Color` so it's not exactly the same as the `toRgb` signature

**Comment by SuperSodaSea** [comment]
Yeah I think this inconsistent is a mistake. We can't change the definition of `source` since the `RgbColor` type itself and logic of `RgbColor` to color is implemented in package `colord`. Maybe changing the signature of `toRgb[a]` would be a safer approach:

```ts
type RgbNormalizedColor = RgbColor;

toRgb(): RgbNormalizedColor
{
    // ...
}
```

**Comment by Zyie** [comment]
For this issue we have updated the documentation to clearly state that `toRgb` will return a normalised color.
So I will close this issue down for now
